src/modules/kpi-reporting/controller.js

```diff
@@ -0,0 +1,50 @@
+'use-strict';
+const helpers = require('@envage/water-abstraction-helpers');
+const dataService = require('./services/data-service');
+
+const Boom = require('@hapi/boom');
+const mappers = require('./lib/mappers');
+
+const isDataNull = (data, errorMessage) => {
```

Comment:
A function that starts with `is` gives the impression a boolean is going to be returns, this might be better named `getErrorMessage`

Reply:
changed as suggested thanks